app/src/main/java/com/khuttun/notificationnotes/NotesListAdapter.java

```diff
@@ -202,6 +202,7 @@ private void setNotification(NotificationNote n)
         if (PreferenceManager.getDefaultSharedPreferences(this.context).getBoolean(this.context
                 .getString(R.string.group_notif_pref_key), false))
         {
+            this.notificationMgr.checkNotification(n);
```

Comment:
I don't see why this would be needed? Changing the group notes setting should update the notifications to reflect the current state (see SettingsActivity).

Reply:
Now multiple notifications with unique ids are created, so when one is unchecked, it does not go away when the notifications are remade.

Reply:
I see. Would it work if you rather called clearAllNotifications in the beginning of displayGroupNotifications? That would seem like a more robust approach.